internal/admission/validator.go

```diff
@@ -70,10 +79,7 @@ func NewKongHTTPValidator(
 // If an error occurs during validation, it is returned as the last argument.
 // The first boolean communicates if the consumer is valid or not and string
 // holds a message if the entity is not valid.
-func (validator KongHTTPValidator) ValidateConsumer(
-	ctx context.Context,
-	consumer kongv1.KongConsumer,
-) (bool, string, error) {
+func (validator KongHTTPValidator) ValidateConsumer(ctx context.Context, consumer kongv1.KongConsumer, operation admissionv1.Operation) (bool, string, error) {
```

Comment:
Can you please add a comment about `operation` above?

Reply:
d270ca6